src/DataFrame/Functions.hs

```diff
@@ -76,28 +76,35 @@ not = Apply "not" Prelude.not
 
 count :: (Columnable a) => Expr a -> Expr Int
 count (Col name) = GeneralAggregate name "count" VG.length
-count _ = error "Argument can only be a column reference not an unevaluated expression"
+count _ = error "count: argument can only be a column reference not an unevaluated expression"
```

Comment:
Actually I have a separate PR in feature engineering addressing these. The aggregates are designed badly. They should take an expression not a column name. 

https://github.com/mchav/dataframe/blob/3177f426dad9dfc0a441c0308b70324fa458c0d9/src/DataFrame/Functions.hs#L86

Maybe a good opportunity to just merge those changes so this isn't throw away work? Wdyt?